tests/integration_tests/with_db/deployment/test_language_container_deployer_cli.py

```diff
@@ -1,36 +1,45 @@
-import ssl
 import textwrap
-from pathlib import Path
+from typing import Optional, Callable
 from urllib.parse import urlparse
 
-import pyexasol
 import pytest
 from click.testing import CliRunner
+from exasol_script_languages_container_tool.lib.tasks.export.export_info import ExportInfo
 from pyexasol import ExaConnection
 from pytest_itde import config
-from pytest_itde.config import TestConfig
 
 from exasol_transformers_extension import deploy
-from tests.utils.db_queries import DBQueries
 from tests.utils.parameters import bucketfs_params
 from tests.utils.revert_language_settings import revert_language_settings
 
 
-@revert_language_settings
-def _call_deploy_language_container_deployer_cli(
-        language_alias: str,
-        schema: str,
-        pyexasol_connection: ExaConnection,
-        version, container_path,
-        language_settings,
-        exasol_config: config.Exasol,
-        bucketfs_config: config.BucketFs):
-    pyexasol_connection.execute(f"DROP SCHEMA IF EXISTS {schema} CASCADE;")
-    pyexasol_connection.execute(f"CREATE SCHEMA IF NOT EXISTS {schema};")
-    dsn = f"{exasol_config.host}:{exasol_config.port}"
+def create_and_run_test_udf(language_alias: str,
+                            schema: str,
+                            connection_factory: Callable[[config.Exasol], ExaConnection],
+                            exasol_config: config.Exasol):
+    # We need a new connection to get the new system value for the SCRIPT_LANGUAGES parameter
+    with connection_factory(exasol_config) as pyexasol_connection:
+        pyexasol_connection.execute(f"OPEN SCHEMA {schema}")
+        pyexasol_connection.execute(textwrap.dedent(f"""
+        CREATE OR REPLACE {language_alias} SCALAR SCRIPT "TEST_UDF"()
+        RETURNS BOOLEAN AS
+    
+        def run(ctx):
+            return True
+    
+        /
+        """))
+        result = pyexasol_connection.execute('SELECT "TEST_UDF"()').fetchall()
+    return result
```

Comment:
```suggestion
        result = pyexasol_connection.execute('SELECT "TEST_UDF"()').fetchall()
        return result
```
Same here: the result should always be the same. Why return it?